src/ViewModel/ArticleSection.php

```diff
@@ -74,9 +78,10 @@ public static function collapsible(
         string $body,
         bool $isInitiallyClosed = false,
         bool $isFirst = false,
-        Doi $doi = null
+        Doi $doi = null,
+        Link $headerLink = null
```

Comment:
Does this work correctly? Can't see an example in Pattern Lab?

Reply:
Is https://github.com/elifesciences/pattern-library/pull/952/files#diff-dc049d04f8fbee547e185f8c3f668925 a sufficient example?